WolvenKit.App/Services/ISettingsManager.cs

```diff
@@ -192,5 +192,7 @@ public static List<SaveGame> GetSaveGames()
 
     bool ReopenLastProject { get; set; }
 
+    int NumFilesToReopen { get; set; }
```

Comment:
There should be an option "Reopen Files on launch" in addition to the number of files to reopen imo